app/assets/javascripts/application.js

```diff
@@ -12,5 +12,6 @@
 //
 //= require jquery
 //= require jquery_ujs
+//= require bootstrap-sprockets
 //= require jasny-bootstrap
```

Comment:
I kept it in. If it just applies to embedded, it can be moved to embedded templates only.